pkg/clients/device/device.go

```diff
@@ -200,23 +210,27 @@ func LateInitialize(in *v1alpha2.DeviceParameters, device *packngo.Device) {
 // from the supplied Packet resource. It considers only fields that can be
 // modified in place without deleting and recreating the instance, which are
 // immutable.
-func IsUpToDate(d *v1alpha2.Device, p *packngo.Device) bool {
+func IsUpToDate(d *v1alpha2.Device, p *packngo.Device) (upToDate bool, networkTypeUpdate bool) {
```

Comment:
in this `IsUpToDate` function, the first result is wether the device is up to date (true is good), and the second result is if the network type needs updating (true is bad?)

Should these both report back a common jitter value? What value would a third isUpToDate response emit?